ui/js/react/treeherder/revision-list.jsx

```diff
@@ -0,0 +1,95 @@
+'use strict';
```

Comment:
I'd agree about doing in another PR except that this PR is the one that moves them to this new location, so I think we should either (a) leave them where they are and restructure later, (b) still move them, but use a more functional directory structure rather than react vs not. Happy with either approach :-)